assets/app/views/add.js

```diff
@@ -44,17 +40,22 @@ var AddSiteView = Backbone.View.extend({
       error: this.onError.bind(this)
     });
   },
+  getFormData: function (sel) {
+    var data = {};
+    this.$(sel).serializeArray().map(function(d) {
+      if (d.name === 'users') d.value = [+d.value];
```

Comment:
I _think_ this project is following the [airbnb](https://github.com/airbnb/javascript) guidelines and I _think_ those guidelines prefer using `{ }` around conditional bodies rather than inlining it. I could not actually find this rule explicitly stated though. _shrugs_.